### PR TITLE
[EKA-000]CM_APP_22:Added Login Activity test

### PR DESCRIPTION
**Pull request by KrupaDesai22**

### CODE REVIEW

File: app/src/androidTest/java/in/projecteka/jataayu/consent/ui/fragment/LoginActivityTest.kt [hunk 1]
```diff
@@ -1,60 +1,50 @@
...
+import br.com.concretesolutions.kappuccino.actions.TextActions
 import org.hamcrest.CoreMatchers.not
-import org.junit.Before
 import org.junit.Rule
 import org.junit.Test
 import org.junit.runner.RunWith
 
-//TODO: REWRITE TO USE LOGIN ACTIVITY
 @RunWith(AndroidJUnit4::class)
 class LoginFragmentTest{
```
Comment (AnusreeSajeevan) [inline app/src/androidTest/java/in/projecteka/jataayu/consent/ui/fragment/LoginActivityTest.kt]: Recheck the class name

Comment (KrupaDesai22) [reply app/src/androidTest/java/in/projecteka/jataayu/consent/ui/fragment/LoginActivityTest.kt]: updated

**Review by AnusreeSajeevan — APPROVED** [review]
Looks good

**Review by AnusreeSajeevan — APPROVED** [review]
Looks good.

**Pull request merged**